### ISSUE TITLE
[Zotero 7] Annotations on HTML snapshots are not in the Data Explorer

### ISSUE THREAD

**Issue by timshadel** [mgmeyers/obsidian-zotero-integration#350]
And they obviously then don't show up in my Obsidian note. The annotations show up in Zotero 7's UI. They don't show in the Data Explorer. No errors are seen in Obsidian's developer console.

| Software | Version |
| --- | --- |
| Zotero | 7.0.0-beta.65 |
| Obsidian Zotero Integration: | 3.1.7 |
| Obsidian: | 1.5.8 |

## Zotero 7 beta

<img width="1290" alt="Screenshot 2024-03-02 at 2 53 18 PM" src="https://github.com/mgmeyers/obsidian-zotero-integration/assets/2394/4b3da35c-c678-4886-8539-1c70c185f19b">

## Zotero Integration - Data Explorer - 3.1.7

<img width="769" alt="Screenshot 2024-03-02 at 2 55 00 PM" src="https://github.com/mgmeyers/obsidian-zotero-integration/assets/2394/1cff8e24-2d86-4e70-9a8c-e8e95fd2b37f">

## Obsidian Console

<img width="838" alt="Screenshot 2024-03-02 at 2 53 45 PM" src="https://github.com/mgmeyers/obsidian-zotero-integration/assets/2394/1b18133b-0273-4a2f-b384-796ff07c1a75">

**Comment by MSHelm** [comment]
I have the same issue. The new website snapshot feature of Obsidian is awesome, I would really love to be able to import the annotations into Obsidian as well.
But probably also to be expected that not all Zotero beta features are already supported by the plugin here :)

**Comment by appwcn** [comment]
same issue here.

**Comment by wpvking** [comment]
Same. The rest of the data seems to be pulling through correctly. Missing Annotations though.

**Comment by rpj-07** [comment]
I have the same issue.

**Comment by abovetheborg** [comment]
I have the same issue and I think it is related to #285 as well

Zotero: 7.0.0-beta.114+d869d7898
BetterBibTex Plugin: 6.7.217
Obsidian: 1.6.7
Zotero Integration: 3.1.7

**Comment by LEEYUEHLU** [comment]
same issue here

**Comment by Lorite** [comment]
I have tested now it seems to be working now.

**Comment by FeralFlora** [comment]
> I have tested it now and it seems to be working.

Yeah, the annotation text is available now, but some metadata like `desktopURI` is still missing, but you can construct a working URI manually based on the parentKey and annotation ID.
